src/main/webapp/profile.jsp

```diff
@@ -11,9 +11,13 @@
     <script src="http://www.w3schools.com/lib/w3data.js"></script>
     <script src="javascript/profile.js"></script>
     <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <script src="javascript/common.js"></script>
```

Comment:
I suggest grouping similar items together, not interleaving them with other items - in this case, placing all references to JS files together (right now they're interrupted by a CSS reference).